src/mumble/Plugins.cpp

```diff
@@ -540,7 +540,7 @@ void Plugins::fetched(QByteArray data, QUrl url) {
 
 	bool rescan = false;
 	const QString &urlPath = url.path();
-	if (urlPath == QLatin1String("/plugins.php")) {
+	if (urlPath == QLatin1String("/v1/pa-plugins")) {
```

Comment:
We should take this chance to split up the fetched slot function used for the list being returned and a download being finished. E.g. ```fetched_update_pa_plugins``` and ```fetched_pa_plugin_dl``` or something.